### PR TITLE
networking: review logs

### PR DESCRIPTION
**Pull request by jmontesi**
The log level and the information printed has been reviewed for each log or added if missing.

The resources are printed between quotes for easier identification in the log line.

### CODE REVIEW

File: cnf-certification-test/networking/icmp/icmp.go [hunk 1]
```diff
@@ -139,27 +141,32 @@ func RunNetworkingTests( //nolint:funlen
...
-			logger.Info("%s ping test on network %s from ( %s  srcip: %s ) to ( %s dstip: %s ) result: %s",
+			logger.Debug("Ping results: %q", result)
+			logger.Info("%q ping test on network %q from ( %q  srcip: %q ) to ( %q dstip: %q ) result: %q",
 				aIPVersion, netName,
 				netUnderTest.TesterSource.ContainerIdentifier, netUnderTest.TesterSource.IP,
-				aDestIP.ContainerIdentifier, aDestIP.IP, result.String())
+				aDestIP.ContainerIdentifier, aDestIP.IP, result)
 			if err != nil {
-				logger.Debug("Ping failed with err:%s", err)
+				logger.Debug("Ping failed, err=%s", err)
```
Comment (rdavid) [inline cnf-certification-test/networking/icmp/icmp.go:162]: Should it be '%q'?

File: cnf-certification-test/networking/netcommons/netcommons.go [hunk 2]
```diff
@@ -149,18 +149,20 @@ func FilterIPListByIPVersion(ipList []string, aIPVersion IPVersion) []string {
...
 }
 
-func FindRogueContainersDeclaringPorts(containers []*provider.Container, portsToTest map[int32]bool, portsOrigin string) (compliantObjects, nonCompliantObjects []*testhelper.ReportObject) {
+func findRogueContainersDeclaringPorts(containers []*provider.Container, portsToTest map[int32]bool, portsOrigin string, logger *log.Logger) (compliantObjects, nonCompliantObjects []*testhelper.ReportObject) {
 	for _, cut := range containers {
+		logger.Info("Testing Container %q", cut)
 		for _, port := range cut.Ports {
 			if portsToTest[port.ContainerPort] {
-				log.Debug("%s has declared a port (%d) that has been reserved", cut, port.ContainerPort)
+				logger.Error("%q declares %s reserved port %d (%s)", cut, portsOrigin, port.ContainerPort, port.Protocol)
```
Comment (rdavid) [inline cnf-certification-test/networking/netcommons/netcommons.go:157]: Should it be Debug?

**Review by rdavid — APPROVED** [review]

File: cnf-certification-test/networking/icmp/icmp.go (hunk 1 above)
Comment (jmontesi) [reply cnf-certification-test/networking/icmp/icmp.go:162]: Let's use `%v` for consistency with the rest of the code.

File: cnf-certification-test/networking/netcommons/netcommons.go (hunk 2 above)
Comment (jmontesi) [reply cnf-certification-test/networking/netcommons/netcommons.go:157]: No. A container declaring a reserved port is an error.

**Review by greyerof — APPROVED** [review]

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**